backend/api/projects.py

```diff
@@ -187,3 +187,190 @@ def update_project(project_id):
                            "the request body does not appear to be utf-8", 400)
     except Exception as err:
         raise ApiException(str(err), 500)
+
+
+@projects.route('/api/projects/<uuid:project_id>/comment', methods=['PUT'])
+@login_required
+def add_comment(project_id):
+    """Adds new comment to project
+
+    Args:
+        project_id, comment in string format
+
+    Returns:
+        response: Success response
+                  or 404 if project is not found
+                  or 400 if validation error occurs
+    """
+    try:
+        manifest = g.projects.find_one({'_id': project_id})
+        if manifest is None:
+            raise ApiException("Project not found", 404)
+        if "text/plain" not in request.content_type:
+            raise ApiException("Comment had not format string", 400)
```

Comment:
yes I'm already working on it and tweaking a few things + implementing current user